src/boot/axios.ts

```diff
@@ -13,17 +13,12 @@ declare module '@vue/runtime-core' {
 // good idea to move this instance creation inside of the
 // "export default () => {}" function below (which runs individually
 // for each client)
-// FIXME: Nested if statements are a code smell, fix it.
```

Comment:
Keep fix me, reword to something like: "FIXME: Hidden if/else chain is a code smell, fix it."